src/main/dto/UserProfileDTO.java

```diff
@@ -0,0 +1,91 @@
+package dto;
+
+import java.sql.Date;
+import constants.Sex;
+
+public class UserProfileDTO{
+    private String account_id;
+    private String full_name;
+    private Sex sex;
+    private String student_code;
+    private String contact_email;
+    private int generation_id;
+    private Date date_of_birth;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              
+    private String userName;
+
+    public UserProfileDTO() {}
```

Comment:
same above

Reply:
I need a default constructor to work with. I init default then set the values for it

Reply:
where did u use it ?